bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -253,6 +275,10 @@ spec:
                   sleep 2
                   done
                 fi
+        readinessProbe:
+          initialDelaySeconds: 30
```

Comment:
Question, I am not sure how readinessProbes impact hostNetwork pods. But will ovnkube-node be able to communicate with its depedencies in ovnkube-master during the first 30 seconds while ovnkube-master is not ready? Otherwise I suspect we might have a lot of errors logged in those pods (if they don't even CrashLoop?)

Reply:
Very good question. In this case, that won't be an issue, because we're not using a Service / Endpoints. Basically, only Ready pods can show up in a Service. However, we talk directly to the pods, so that's not an issue here.